REUSE.toml

```diff
@@ -0,0 +1,52 @@
+version = 1
+
+[[annotations]]
+path = [
+  ".check.exs",
+  ".doctor.exs",
+  ".formatter.exs",
+  ".github/dependabot.yml",
+  ".github/workflows/elixir.yml",
+  ".gitignore",
+  ".tool-versions",
+  ".vscode/settings.json",
+  "CHANGELOG.md",
+  "CLAUDE.md",
+  "NOTICE",
+  "README.md",
+  "REUSE.toml",
+  "config/config.exs",
+  "documentation/.gitkeep",
+  "documentation/dsls/DSL-Reactor.md",
+  "documentation/explanation/architecture.md",
+  "documentation/explanation/concepts.md",
+  "documentation/explanation/design-decisions.md",
+  "documentation/explanation/ecosystem.md",
+  "documentation/how-to/api-orchestration.md",
+  "documentation/how-to/data-pipelines.md",
+  "documentation/how-to/debugging-workflows.md",
+  "documentation/how-to/payment-processing.md",
+  "documentation/how-to/performance-optimization.md",
+  "documentation/how-to/testing-strategies.md",
+  "documentation/reference/glossary.md",
+  "documentation/tutorials/01-getting-started.md",
+  "documentation/tutorials/02-error-handling.md",
+  "documentation/tutorials/03-async-workflows.md",
+  "documentation/tutorials/04-composition.md",
+  "documentation/tutorials/05-recursive-execution.md",
+  "documentation/tutorials/reactor-cheatsheet.cheatmd",
+  "logos/reactor-logo-dark-medium.png",
+  "logos/reactor-logo-dark-small.png",
+  "logos/reactor-logo-dark.png",
+  "logos/reactor-logo-dark.svg",
+  "logos/reactor-logo-light-medium.png",
+  "logos/reactor-logo-light-small.png",
+  "logos/reactor-logo-light.png",
+  "logos/reactor-logo-light.svg",
```

Comment:
Use `[PATH].license` files for things that you can't comment. See https://reuse.software/faq/#uncommentable-file